src/controllers/start.controller.ts

```diff
@@ -1,12 +1,20 @@
 import { Request, Response } from "express";
-import { CHS_URL, PIWIK_START_GOAL_ID, FEATURE_FLAG_REMOVE_DIRECTOR_20022023, EWF_URL, AP01_ACTIVE } from "../utils/properties";
+import {
+  CHS_URL,
+  PIWIK_START_GOAL_ID,
+  FEATURE_FLAG_REMOVE_DIRECTOR_20022023,
+  EWF_URL,
+  AP01_ACTIVE,
+  CH01_ACTIVE
+} from "../utils/properties";
 import { Templates } from "../types/template.paths";
 
 export const get = (req: Request, res: Response) => {
   return res.render(Templates.START, { CHS_URL,
     PIWIK_START_GOAL_ID,
     FEATURE_FLAG_REMOVE_DIRECTOR_20022023,
     AP01_ACTIVE,
+    CH01_ACTIVE,
```

Comment:
Do we need this here? Might be left over from testing

Reply:
yes I did noticed that, however I believe it's going to be useful to display the content of the [start ](https://github.com/companieshouse/officer-filing-web/blob/main/views/start.html#L16-L23) page.